app/assets/stylesheets/custom.scss

```diff
@@ -14,13 +14,10 @@ h1 {
   margin-bottom: 40px;
 }
 
-.form-title {
-}
-
 #title {
-  width: 103%;
+  width: 100%;
```

Comment:
Properties should be ordered background-color, font-weight, margin, padding, width